lib/diagnostic.rb

```diff
@@ -78,10 +93,12 @@ def initialize
 
 ## What will be the output from calling `HouseCat.new.say_hello`?
 # replace nil with your response
-Response.housecat_noise = nil
+Response.housecat_noise = "I am a HouseCat, and I go 'meow'"
 
 ## Explain why this would be the output, based on the method lookup chain.
-# your response as a comment here
```

Comment:
`say_hello` is not defined on `HouseCat` or `Cat`, but it is defined on `Animal`, so it needs be looked up in its superclasses.